src/bench/rpc_blockchain.cpp

```diff
@@ -15,25 +15,49 @@
 
 #include <univalue.h>
 
-static void BlockToJsonVerbose(benchmark::Bench& bench)
-{
+namespace {
+
+struct TestBlockAndIndex {
     TestingSetup test_setup{};
+    CBlock block{};
+    uint256 blockHash{};
+    CBlockIndex blockindex{};
 
-    CDataStream stream(benchmark::data::block813851, SER_NETWORK, PROTOCOL_VERSION);
-    char a = '\0';
-    stream.write(&a, 1); // Prevent compaction
+    TestBlockAndIndex()
+    {
+        CDataStream stream(benchmark::data::block813851, SER_NETWORK, PROTOCOL_VERSION);
+        char a = '\0';
+        stream.write(&a, 1); // Prevent compaction
 
-    CBlock block;
-    stream >> block;
+        stream >> block;
 
-    CBlockIndex blockindex;
-    const uint256 blockHash = block.GetHash();
-    blockindex.phashBlock = &blockHash;
-    blockindex.nBits = 403014710;
+        blockHash = block.GetHash();
+        blockindex.phashBlock = &blockHash;
+        blockindex.nBits = 403014710;
+    }
+};
 
+} // namespace
+
+static void BlockToJsonVerbose(benchmark::Bench& bench)
+{
+    TestBlockAndIndex data;
     bench.run([&] {
-        (void)blockToJSON(block, &blockindex, &blockindex, *llmq::chainLocksHandler, *llmq::quorumInstantSendManager, /*verbose*/ true);
+        auto univalue = blockToJSON(data.block, &data.blockindex, &data.blockindex, *llmq::chainLocksHandler, *llmq::quorumInstantSendManager, /*verbose*/ true);
```

Comment:
we have chainLocksHandler and quorumInstantSendManager in testing environment.
Instead using global wariables, let's get access to them throught `test_setup->node->llmq_ctx->`

Reply:
Changes could be done in a separate PR, not within the scope of the backport itself.